lib/improver/utilities/cube_manipulation.py

```diff
@@ -211,14 +210,17 @@ def concatenate_cubes(
     return result
 
 
-def merge_cubes(cubes):
+def merge_cubes(cubes, model_id_attr=None):
     """
     Function to merge cubes, accounting for differences in the
     attributes, and coords.
 
     Args:
         cubes (Iris cubelist or Iris cube):
             Cubes to be merged.
+        model_id_attr (str):
+            Identification string of the model configuration, required for
+            building a Model ID coord if grid blending cubes.
```

Comment:
Again, could you change to `Name of cube attribute used to identify the model for grid blending.`

Reply:
Done.